devon4j.properties

```diff
@@ -0,0 +1,14 @@
+category=Governance
+description=Plugin for SonarQube to validate devon4j architecture and conventions.
```

Comment:
The "Plugin for SonarQube to" part is a bit redundant in this context, but okay.